package.json

```diff
@@ -117,7 +117,12 @@
                 "scopeName": "source.atf",
                 "path": "./syntaxes/atf.tmLanguage.json"
             }
-        ]
+        ],
+	"configurationDefaults": {
+	    "[atf]": {
+		"editor.wordWrap": "on"
+	    }
+	}
```

Comment:
This renders strangely on the diff viewer, looks like there's a mixture of tabs and spaces? 🙀

Reply:
Right, I literally copied these lines from the markdown extension, I changed to all spaces.